instance/management/commands/activity_csv.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2016 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance app - Activity report management command
+"""
+
+# Imports #####################################################################
+
+from configparser import ConfigParser
+import csv
+from datetime import datetime
+import os
+import sys
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from instance import ansible
+from instance.models.openedx_instance import OpenEdXInstance
+from instance.utils import poll_streams
+
+
+# Classes #####################################################################
+
+class Command(BaseCommand):
+    """
+    Activity_csv management command class
+    """
+    help = (
+        'Generates a CSV containing basic activity information about active instances'
```

Comment:
Done. Yeah, thanks for catching things like this. As you can probably tell I had some trouble with terminology. I wouldn't be surprised if I confused "instance" with "app servers" in a few other places as well.